src/globalnotifier.js

```diff
@@ -45,11 +45,19 @@ wrapper.init = function(config, parent) {
 
   if (config.captureUncaught) {
     // Set the global onerror handler
-    var old = window.onerror;
+    var oldOnError;
+
+    // If the parent, probably a shim, stores a oldOnError, use that so we don't
+    // send reports twice.
+    if (parent && typeof parent.oldOnError !== 'undefined') {
```

Comment:
Would it make sense to put 'rollbar' in that variable name somewhere, to make sure it really was us? e.g. `parent._rollbar_oldOnError` or something.